vms/vms/urls.py

```diff
@@ -16,4 +22,7 @@
     url(r'^registration/', include('registration.urls', namespace='registration')),
     url(r'^shift/', include('shift.urls', namespace='shift')),
     url(r'^volunteer/', include('volunteer.urls', namespace="volunteer")),
+    url(r'^portal', TemplateView.as_view(template_name='home/home.html'), name='home'),
+    url(r'^login/$', anonymous_required(auth_views.login), {'template_name': 'authentication/login.html'}, name='login_process'),
```

Comment:
@amruthasangeeth I think urls are changing here. They have been shifted from the authentication app to the main urls.py file. Please recheck.